### PR TITLE
doc: add comments related to actor code review

### PR DESCRIPTION
**Pull request by schomatis**
Feel free to suggest or change anything here, the objective of this PR is to signal some places that could use a comment.

### CODE REVIEW

File: chain/actors/actor_init.go [hunk 1]
```diff
@@ -42,6 +42,9 @@ func init() {
 type InitActor struct{}
 
 type InitActorState struct {
+	// Maps generic `Address` to its `ID` type.
+	// FIXME: Q: Is this any address type or just `Actor`? (see `InitActor.Exec`).
+	//  Through `RegisterNewAddress` seems that it can be any address.
```
Comment (schomatis) [inline chain/actors/actor_init.go]: Question.

**Comment by schomatis** [comment]
I'm not sure what's the `mod-tidy-check` error about.

**Comment by Kubuxu** [comment]
> I'm not sure what's the mod-tidy-check error about.

Run `go mod tidy`.

**Comment by schomatis** [comment]
> probably someone else who left the branch is not tidy state.

Yeah, these are just comments, didn't change anything module-related.

**Comment by schomatis** [comment]
@magik6k rebased on top testnet, can we get this in or should I change something else?

**Pull request merged**